kratos/tests/test_normal_utils_mpi.py

```diff
@@ -0,0 +1,170 @@
+from __future__ import print_function, absolute_import, division
```

Comment:
@loumalouomega you misunderstood a bit how the testing in MPI is done

The point is that THE SAME test has to be run in order to make sure the results are consistent in shared and distributed memory => the only difference is the reading of the ModelPart

Take a look at how @sunethwarna has done it in #7345

I will take a closer look so that this can go in when I am back from vacations in two weeks

Reply:
Well,  @sunethwarna is going to fix it (I am still on vacation). The important thing is that the utilities work in MPI, it is just a matter of adapting the test

Reply:
done :)

Reply:
Hurray